packages/xdl/src/Versions.js

```diff
@@ -83,6 +83,16 @@ export async function newestSdkVersionAsync() {
   return result;
 }
 
+export async function nextDeprecatedSdkVersionAsync() {
```

Comment:
This function doesn't return the full semver version, but the major version part of it. Maybe make it return the whole semver version (preferred) or change the function name to mention it returns a major version (e.g. `oldestSupportedMajorVersionAsync`).

Reply:
You can use `semver.compare` as a compare function passed to `Array.prototype.sort` to sort by semver order.